pkg/services/searchV2/index.go

```diff
@@ -91,9 +98,9 @@ func (i *dashboardIndex) run(ctx context.Context) error {
 		lastEventID = lastEvent.Id
 	}
 
-	err = i.buildInitialIndex(ctx)
+	err = i.buildInitialIndexes(ctx, orgIDs)
```

Comment:
What do you think about sending a "build signal" for each org here, rather than building each org's index in a sequence?


1. App start -> `buildSignal(1), buildSignal(2), buildSignal(3), ...`
2. Indexes are built asynchronously - we might need to limit the # number of indexes building at the same time
3. Read requests are blocking until their corresponding org's index has been built

The main advantage I see is 
- no slowdown on startup for essentially the same experience in _most_ cases
- no cascading failure - currently looong/stuck index rebuild for org=1 blocks other orgs from working properly

Reply:
I think it's a good optimization it terms of faster startup time for some orgs - so as soon as org indexed we can serve it. But not sure we want to parallelize index creation due to rather aggressive resource usage – so most probably we will index one by one anyway.

At the moment we are not indexing asynchronously (it's done in another PR) – so let's try after combining everything together?

Reply:
Sounds good!